integration-tests/contracts/FakeOptimismMintableERC721.sol

```diff
@@ -2,6 +2,8 @@
 pragma solidity ^0.8.9;
 
 import { ERC721 } from "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import { IOptimismMintableERC721 } from "@eth-optimism/contracts-periphery/contracts/universal/op-erc721/IOptimismMintableERC721.sol";
+import { IERC165 } from "@openzeppelin/contracts/utils/introspection/IERC165.sol";
 
 contract FakeOptimismMintableERC721 is ERC721 {
```

Comment:
Shouldn't this inherit from `OptimismMintableERC721` and have almost no implementation logic? 
Ie. the `safeMint` and `supportsInterface` functions are already on `OptimismMintableERC721`

Reply:
Yeah, fixed this. I left the `safeMint` and `burn` function as-is though because the overridden contract has an `onlyBridge` modifier, which we need to remove in the overriding contract.